RaidBot/Modules/RaidModule.cs

```diff
@@ -248,8 +274,22 @@ public class RaidModule : ModuleBase {
          }
       }
 
+      [Command("Go"), Summary("Informs everyone in a raid that the raid is starting")]
+      [Alias("Start", "Lobby")]
+      public async Task Go([Summary("The name of the raid")] string raidName) {
+         var requesterUser = Context.User as IGuildUser;
+         var result = _raidService.MessageUsers(raidName, $"The raid is starting now. Please go to the lobby");

```

Comment:
Is this something we want only the leader to be able to do?

Reply:
I thought about it, but this is simply a shortcut for `$message raidName Let's jump in`, which today anyone can do. Messaging everyone in a raid is not restricted to the leader - we can leave `$go` like this as well and see how it goes. In practice, if someone tells everyone to jump in mistakenly this will be noticed and people will probably tell the person not to do this again anyway.